runtime/src/lib.rs

```diff
@@ -1010,29 +1085,56 @@ impl common::StorageOwnership for Runtime {
 // Storage parameters independent of runtime profile
 parameter_types! {
     pub const MaxDistributionBucketFamilyNumber: u64 = 200;
-    pub const BlacklistSizeLimit: u64 = 10000; //TODO: adjust value
-    pub const MaxNumberOfPendingInvitationsPerDistributionBucket: u32 = 20; //TODO: adjust value
+    pub const BlacklistSizeLimit: u64 = 1_000;
+    pub const MaxNumberOfPendingInvitationsPerDistributionBucket: u32 = 20;
     pub const StorageModuleId: PalletId = PalletId(*b"mstorage"); // module storage
-    pub const MinDistributionBucketsPerBag: u32 = 1; //TODO: adjust value
-    pub const MaxDistributionBucketsPerBag: u32 = 100; //TODO: adjust value
-    pub const MaxDataObjectSize: u64 = 10 * 1024 * 1024 * 1024; // 10 GB
+    pub const MinDistributionBucketsPerBag: u32 = 1;
+    pub const MaxDistributionBucketsPerBag: u32 = 51;
+    pub const MaxDataObjectSize: u64 = giga_bytes!(30);
```

Comment:
Can we double this to 60?

Reply:
Sure

Reply:
Done in https://github.com/Joystream/joystream/pull/4341/commits/2adcc04e3f6aa5b615f9dda4696fe93e5aac93ee